WordPress/Classes/ViewRelated/Site Creation/DesignSelection/SiteDesignContentCollectionViewController.swift

```diff
@@ -1,67 +1,78 @@
 import UIKit
 import WordPressKit
 
-class SiteDesignContentCollectionViewController: CollapsableHeaderViewController, UIPopoverPresentationControllerDelegate {
+class SiteDesignSection: CategorySection {
```

Comment:
I had this in mind at the start of the refactor, but if I understand correctly, I think some other things need to be addressed before we can do this, because we rely on the more specific type bindings throughout the subclass implementations.

For example, if we renamed `designs` and `layouts` to some common name in a struct, with a protocol type, and eliminated the type heirarchy of the `sections`, we can no longer safely dereference the needed properties / methods from the data model instances. We could address that by merging the data models too (these endpoints are now looking nearly identical), but that feels inflexible with respect to API changes, and also limits the usage to things conforming to that particular unified data model.